src/Common/RecordingTracingInterceptor.cs

```diff
@@ -19,18 +19,23 @@
 using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Net.Http;
+using System.Text.RegularExpressions;
 
 namespace Microsoft.Azure.ServiceManagement.Common.Models
 {
     public class RecordingTracingInterceptor : Hyak.Common.ICloudTracingInterceptor
     {
-        public RecordingTracingInterceptor(ConcurrentQueue<string> queue)
+
+        public RecordingTracingInterceptor(ConcurrentQueue<string> queue, IList<Regex> matchers = null)
```

Comment:
Keep original constructor to ensure capability

Reply:
this will be compatible with the original constructor